src/main/io/ledstrip.c

```diff
@@ -1321,23 +1321,23 @@ bool parseColor(int index, const char *colorConfig)
 /*
  * Redefine a color in a mode.
  * */
-bool setModeColor(ledModeIndex_e modeIndex, int modeColorIndex, int colorIndex)
+bool setModeColor(ledFCStates_e fcStatusIndex, int modeColorIndex, int colorIndex)
 {
     // check color
     if (colorIndex < 0 || colorIndex >= LED_CONFIGURABLE_COLOR_COUNT)
         return false;
-    if (modeIndex < LED_MODE_COUNT) {  // modeIndex_e is unsigned, so one-sided test is enough
+    if (fcStatusIndex < LED_FCSTATE_COUNT) {  // ledFCStates_e is unsigned, so one-sided test is enough
```

Comment:
`enum`s may be signed (it's implementation-defined). 
```suggestion
    if (fcStatusIndex >= 0 && fcStatusIndex < LED_FCSTATE_COUNT) {
```

Reply:
thanks, again

Reply:
Failed the build test so for now am reverting this one.